# This removes the vendored googletest dependency. EasyBuild uses the module instead
# Author : Maxime Boissonneault (Compute Canada)
diff -ru sdsl-lite-c32874cb2d8524119f25f3b501526fe692df29f4.orig/external/CMakeLists.txt sdsl-lite-c32874cb2d8524119f25f3b501526fe692df29f4/external/CMakeLists.txt
--- sdsl-lite-c32874cb2d8524119f25f3b501526fe692df29f4.orig/external/CMakeLists.txt	2020-08-10 19:07:11.804540614 -0000
+++ sdsl-lite-c32874cb2d8524119f25f3b501526fe692df29f4/external/CMakeLists.txt	2020-08-10 19:07:34.993778036 -0000
@@ -13,16 +13,4 @@
 
 add_subdirectory(libdivsufsort)
 
-## Add gtest
-set(gtest_dir ${CMAKE_CURRENT_LIST_DIR}/googletest)
-set(gtest_file ${gtest_dir}/CMakeLists.txt)
-if(NOT EXISTS ${gtest_file})
-    EXECUTE_PROCESS(COMMAND git submodule update --init external/googletest
-                    WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/..
-                    OUTPUT_QUIET
-            )
-endif(NOT EXISTS ${gtest_file})
-
-add_subdirectory(googletest/googletest)
-
 
